runtime/src/iree-amd-aie/driver/xrt-lite/direct_command_buffer.cc

```diff
@@ -191,10 +194,22 @@ static iree_status_t iree_hal_xrt_lite_direct_command_buffer_dispatch(
     ebuf.add_arg_bo(*bo);
   }
 
+  auto time0 = std::chrono::high_resolution_clock::now();
+
```

Comment:
it's ugly but i would guard these too ie
```suggestion
#ifdef IREE_AMDAIE_TIME_KERNEL
  auto time0 = std::chrono::high_resolution_clock::now();
#endif
```

Reply:
I tried like 100 variants on what to guard, and decided that keeping guards to the minimum was priority